git-exercises/git-exercise/sayHello.js

```diff
@@ -0,0 +1,4 @@
+function sayHello() {
+	return 'Hello World';
```

Comment:
@abishekbardewa Why this file is included in the Git exercise?

Please remove this file!